src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_fullyconnected.hpp

```diff
@@ -123,8 +123,7 @@ class DnnlFCExecutor : public Executor {
         if (currentPrimitive && currentPrimitive->weightsDesc()->isCompatible(*newPrimMemDesc))
             return;
 
-        if (m_attrs.weightsNonTransposed)
-            originalMemDesc = utils::makeTransposedWeightDescriptor(originalMemDesc, newPrimMemDesc);
+        originalMemDesc = Primitive::makeTransposedWeightDescriptor(originalMemDesc, newPrimMemDesc, m_attrs.weightsNonTransposed);
```

Comment:
Now it is Primitive responsibility to provide proper transposed weight descriptor.
@todo remove makeTransposedWeightDescriptor from dnnl_utils